source/jsonnet/england-wales/household/blocks/who-lives-here/anyone_else.jsonnet

```diff
@@ -0,0 +1,177 @@
+local rules = import '../../../lib/common_rules.libsonnet';
+local placeholders = import '../../lib/placeholders.libsonnet';
+
+local questionTitle(isPrimary) = (
+  if isPrimary then {
+    text: 'Do any of the following people live at {household_address} on {census_date}?',
+    placeholders: [
+      placeholders.address,
+      placeholders.censusDate,
+    ],
+  } else {
+    text: 'Do any of the following people also live at {household_address} on {census_date}?',
+    placeholders: [
+      placeholders.address,
+      placeholders.censusDate,
+    ],
+  }
+);
+
+local anyoneElseAnswerExclusiveText(isPrimary) = (
+  if isPrimary then {
+    label: 'None of these apply, I am the only person who usually lives here',
+    value: 'None of these apply, I am the only person who usually lives here',
+  } else {
+    label: 'None of these apply, no-one usually lives here',
+    value: 'None of these apply, no-one usually lives here',
+    description: 'For example, this is a second address or holiday home',
+  }
+);
+
+local question(isPrimary) = {
+  type: 'MutuallyExclusive',
+  id: 'anyone-else-driving-question',
+  title: questionTitle(isPrimary),
+  definitions: [
+    {
+      title: 'Who to include',
+      contents: [
+        { description: 'This is often their permanent or family home.' },
+        {
+          description: {
+            text: 'If someone has more than one home address, include them at this address if they generally spend most of their time here. For example, children whose parents live apart. If they split their time equally than include them at this address if they are staying overnight on {census_date}.',
```

Comment:
You have a 'than' instead of a 'then'. Definitely not you, but the punctuation on this isn't great.

Reply:
> Your quick launch will naturally session out, so I couldn't use it

works fine for me in a private and standard tab